resources/lib/objects/kodi/queries_music.py

```diff
@@ -82,7 +82,7 @@
     					"""
 add_single_obj =            [   "{AlbumId}","{Genre}","{Year}","single"
                             ]
-add_song =	     		"""	INSERT INTO song(idSong, idAlbum, idPath, strArtists, strGenres, strTitle, iTrack, 
+add_song =	     		"""	INSERT INTO song(idSong, idAlbum, idPath, strArtistDisp, strGenres, strTitle, iTrack, 
```

Comment:
what happens for Kodi < 18.1?

Reply:
Can your create an objects/Leia branch, like the emby for kodi git?
Seems this is their “solution”.
I’m quite new to kodi addon dev, I presume the next step is having repo’s with different versions.
Did these changes to get jellyfin working on my Kodi, so didn’t think ahead :P

Reply:
Welp, seems https://github.com/jellyfin/jellyfin-kodi/commit/5fba6555a8ee8e3acbfee7bf79fb571c058bf52b#diff-e986bc05edb125aca769304dee76ab30 had a bunch of code which checked the Kodi version and applied the correct objects patch.
We'll have to bring that back

Reply:
Not to dismiss their work, but they are misusing Git branches and we will not be following that model

Reply:
This seems much saner: https://forum.kodi.tv/showthread.php?tid=310315

Reply:
Indeed.
How do you propose to handle multiple versions of the addon itself?

Reply:
I have no experience with Kodi plugin development, but I was thinking a branch per Kodi version. Barring any major restructuring in the server API, there shouldn't be many discrepancies between Kodi versions that are non-Kodi version specific (if that makes sense).

Reply:
Can we just make runtime checks for Kodi versions? Python is very flexible.
This might be easier than maintain two separate branches.

Reply:
I'm not a big fan of the Emby model where it automatically updates the 'objects' without consent.